src/backend/core/tests/test_models_documents.py

```diff
@@ -1168,184 +1168,135 @@ def test_models_documents_restore_complex_bis(django_assert_num_queries):
 
 
 @pytest.mark.parametrize(
-    "ancestors_links, select_options",
+    "reach, role, select_options",
     [
         # One ancestor
```

Comment:
```suggestion
```

Or maybe `with ancestors` in opposition to `no ancestor` few lines after ?

Reply:
I removed it as this comment does not make sense anymore after the refactorings.